TWLight/resources/tasks.py

```diff
@@ -0,0 +1,26 @@
+from django.core.cache import cache
```

Comment:
So the idea I was trying to get across originally about cache invalidation wasn't that deleting items from the cache needed to be done asynchronously. cache deletion happens in microseconds. I was trying to communicate that we might need to move any expensive queries/network calls required for *cache invalidation logic* out to be asynchronous. However if doing the requests for our cache invalidation logic turns out to be just as expensive as requesting the content, we should replace our concept of invalidation with updating. We could set our ttl to be forever (so that we always have something in the cache) and then go out and fetch a fresh value asynchronously after a reasonable amount of time has passed.